docs/eventing/broker/README.md

```diff
@@ -31,7 +31,7 @@ Before you can use the MT channel-based broker, you must install a
 
 ### Alternative broker implementations
 
-In the Knative Eventing ecosystem, alternative broker implementations are welcome as long as they respect the [broker specifications](https://github.com/knative/specs/blob/main/specs/eventing/broker.md).
+In the Knative Eventing ecosystem, alternative broker implementations are welcome as long as they respect the [broker specifications](https://github.com/knative/specs/blob/f3dcd525/specs/eventing/control-plane.md#broker-lifecycle).
```

Comment:
Would it be better to link to main?

Reply:
@snneji done in [88df8703..fd2b6d1c](https://github.com/knative/docs/compare/88df870321480e5bfa0cd84e96699cb7eccf0081..fd2b6d1c3294463897feb31f008873614043a1fb)
Thanks for the review! 🙌